### PR TITLE
backend/feat: added customer client info with first ride case

### PR DESCRIPTION
**Pull request by ashishsaini25**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

added customer client info with first ride cas

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by vijaygupta18 — DISMISSED** [review]

**Review by piyushKumar-1 — DISMISSED** [review]

File: Backend/dev/migrations/rider-app/1256-add-client-id.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+------ these migration are according to prod--------------------
+insert atlas_app.client (id, short_id, merchant_id, created_at, updated_at) values ('995ff758-4efa-4d18-8f8f-f779521eb743', 'NAMMA_YATRI','4b17bd06-ae7e-48e9-85bf-282fb310209c',now(),now());
+insert atlas_app.client (id, short_id, merchant_id, created_at, updated_at) values ('cab98477-f759-4467-b936-f6759453bc0b', 'JATRI_SAATHI','bd064716-ae7e-48e9-85bf-282fb310209c',now(),now());
+insert atlas_app.client (id, short_id, merchant_id, created_at, updated_at) values ('c3784e1b-c092-4e97-8175-0e5fffaefc44', 'YATRI','c9811842-d572-11ed-afa1-0242ac120002',now(),now());
+insert atlas_app.client (id, short_id, merchant_id, created_at, updated_at) values ('a791920b-8271-4536-bd6c-14bd4e329812', 'MANA_YATRI','4b17bd06-ae7e-48e9-85bf-282fb310209c',now(),now());
+CREATE INDEX idx_client_person_info_person_id ON atlas_rider_app.client_person_info USING btree (person_id);
```
Comment (ashishsaini25) [inline Backend/dev/migrations/rider-app/1256-add-client-id.sql:7]: add unique constraint.

File: Backend/dev/migrations/dynamic-offer-driver-app/0399-add-average-speed-for-bike.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+UPDATE atlas_driver_offer_bpp.transporter_config
+SET avg_speed_of_vehicle = '{"sedan": 30, "suv": 30, "hatchback": 0,"autorickshaw": 0, "taxi" : 0, "taxiplus" : 0, "bike" : 25}'::json where merchant_id='favorit0-0000-0000-0000-00000favorit'; -- replace it only for yathri_sathi merchant_id
```
Comment (ashishsaini25) [inline Backend/dev/migrations/dynamic-offer-driver-app/0399-add-average-speed-for-bike.sql:2]: Add proper commented queries.

File: Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs [hunk 3]
```diff
@@ -181,6 +186,18 @@ getPersonDetails (personId, _) mbToss = do
   frntndfgs <- if useCACConfig then getFrontendConfigs person mbToss else return $ Just DAKM.empty
   let mbMd5Digest = T.pack . show . MD5.md5 . DA.encode <$> frntndfgs
   isSafetyCenterDisabled_ <- SLP.checkSafetyCenterDisabled person
+  hasTakenCabRide <- QCP.findByPersonIdAndVehicleCategory personId $ Just BecknConfig.CAB
```
Comment (ashishsaini25) [inline Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs:189]: find all then filter.

File: Backend/dev/migrations/rider-app/1256-add-client-id.sql (hunk 1 above)
Comment (hkmangla) [reply Backend/dev/migrations/rider-app/1256-add-client-id.sql:7]: Still unresolved.

**Review by hkmangla — CHANGES_REQUESTED** [review]

**Review by hkmangla — APPROVED** [review]

**Pull request merged**